README.md

```diff
@@ -78,27 +78,32 @@ However, many IDEs can be configured to auto format on save.
 
 ## Proposed Reimbursement Work Flow
 
-The proposed workflow is to be automated and run on a schedule of once per _accounting
-period_ at N hours past the end of the accounting period. Note that we must wait some
-time after the period has ended for some data to finalize (e.g. `dex.trades`
-, `ethereum.transactions` our event data, etc...).
+The solver reimbursements are to be executed each Tuesday with the accounting period of the last 7 days. If the payout can not be done on Tuesdays, its okay to execute them later, but still the accounting period should always be from 00:00 between Monday and Tuesday to 00:00 between Monday and Tuesday
```

Comment:
I was under the impression we were going to be doing Monday (beginning) to Sunday (end) (= Monday beginning), but this is fine too.

Reply:
First of March is a tuesday, hence we start with tuesdays. That is how I understood the proposal